x-pack/legacy/plugins/lens/public/help_menu_util.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { EuiHorizontalRule, EuiSpacer, EuiLink, EuiText, EuiIcon, EuiButton } from '@elastic/eui';
+import { ELASTIC_WEBSITE_URL, DOC_LINK_VERSION } from 'ui/documentation_links';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { render, unmountComponentAtNode } from 'react-dom';
+import { Chrome } from 'ui/chrome';
+
+const docsPage = undefined;
```

Comment:
The naming convention seems to be `lens.html` (e.g. `canvas.html` is canvas' landing page). So I suggest we make this `lens.html`, and if that turns out to be wrong, we can fix it later. If it turns out to be right, we're golden.

Reply:
Since this page doesn't exist, I'm not sure we want to link to it already. But I see your point. This could be a thing we do later